src/components/CPU/BreakpointPanelItem.css

```diff
@@ -0,0 +1,13 @@
+.BreakpointPanelItem--selected {
+	background: linear-gradient(to bottom, #39f, #39f);

```

Comment:
This is fine, but it could also be a static color.  The reason I used a linear-gradient in the disasm was honestly a slight hack - it makes it an image.  That way, I can use inline styling from JS for the background shading, and it won't override.

-[Unknown]